index.html

```diff
@@ -1620,7 +1620,7 @@ <h1>Download</h1>
 								<div><span>RAM:</span>4 GiB - 16 GiB LPDDR4</div>
 								<div><span>USB:</span>USB 3.0 OTG and HOST</div>
 								<div><span>NET:</span>2.5 Gbit Ethernet, optional Radxa Wireless Module</div>
-								<div><span>Storage:</span>microSD and eMMC sockets</div>
+								<div><span>Storage:</span>microSD, eMMC and M.2 sockets</div>
```

Comment:
ROCK 5A has an E-key M.2 connector only, not suitable for an SSD, but e.g. for a WiFi/BT module.
```suggestion
								<div><span>Storage:</span>microSD and eMMC sockets</div>
```